### ISSUE TITLE
update of digital identity: connection from DIdentity backend to lissi backend established. Change process triggered

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#27]
## User story
1. As a {HR employee}
2. I want / need {that after submiting changes on frontend, process of editing the digital identity in the lissi backend is triggered}
3. So that {editing of digital identity is successfull}

## Acceptance criteria
* connection to lissi backend established
* update process triggered in lissi backend
* SD has researched how the process of updating a digital identity continues after request has been sent to lissi backend: Does the customer (to whom the digital identity belongs) need to do anything in the process?

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
